docs/components/modeler/bpmn/message-events/message-events.md

```diff
@@ -73,9 +73,9 @@ To correlate a message to the message event, the message is published with the d
 
 ## Variable mappings
 
-By default, all message variables are merged into the process instance. This behavior can be customized by defining an output mapping at the message catch event.
-
-Visit the documentation regarding [variable mappings](/components/modeler/bpmn/service-tasks/service-tasks.md#variable-mappings) for more information on this topic.
+Define [variable mappings](/components/concepts/variables.md#inputoutput-variable-mappings)
+the [same way as a service task does](/components/modeler/bpmn/service-tasks/service-tasks.md#variable-mappings)
+to transform the variables passed to the job worker, or to customize how the variables of the job merge.
```

Comment:
❌ I think that the sentence "same way as a service task does' makes sense for the tasks that are like a service task (`send task`, `script task`, `business rule task`), but it doesn't make much sense for message events, nor for receive tasks because they are not service tasks. They don't have a job worker.

Reply:
Thanks for the hint! I just fixed it.